### PR TITLE
ADD undo rebase/merge

### PR DESCRIPTION
**Pull request by gaetan-petit**
Please correct my broken english :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -561,6 +562,15 @@ To delete a local branch:
 
 ## Rebasing and Merging
 
+<a name="undo-rebase">
+### I want to undo rebase/merge
+
+You may have merge/rebase your current branch with a wrong branch, or you simply cannot manage to achieve a correct result or even finish the rebase/merge process. Git is saving the original HEAD pointer in ORIG_HEAD, so it is simple to recover your branch at the state before the rebase/merge.
```
Comment (RichardLitt) [inline README.md]: > You may have merged or rebased your current branch with a wrong branch, or you can't figure it out or finish the rebase/merge process. Git saves the original HEAD pointer in a variable called ORIG_HEAD before doing dangerous operations, so it is simple to recover your branch at the state before the rebase/merge.

This reads a bit better. Make sense?

Comment (RichardLitt) [inline README.md]: @gaetan-petit Do you want to add this to the PR and edit it? Or do you want me to?

**Comment by gaetan-petit** [comment]
@RichardLitt I updated my PR 😃

**Comment by RichardLitt** [comment]
Thank you!

**Pull request merged**